### PR TITLE
Add OFFLINE migrations

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Addresses ENG-118

Signed-off-by: Vadim Markovtsev <vadim@athenian.co>

### CODE REVIEW

**Comment by vmarkovtsev** [comment]
@lwsanty Does this work for you or there is smth extra/different that you need?

File: server/athenian/api/models/state/README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ python3 -m athenian.api.models.state postgres://user:password@host:port/database
 Replace `postgres://user:password@host:port/database` with the actual DB connection string in
 [SQLAlchemy format](https://docs.sqlalchemy.org/en/13/core/engines.html).
 
+It is possible to define `OFFLINE` environment variable (to any non-empty value) so that
+the SQL operations dump is written to the standard output while the actual DB is stays untouched.
```
Comment (lwsanty) [inline server/athenian/api/models/state/README.md]: maybe I have some problems with English today, but got a nitpick:
> while the actual DB is stays untouched.

`while the actual DB stays untouched.` or `while the actual DB is staying untouched.`

**Comment by lwsanty** [comment]
@vmarkovtsev let's try it with this format first :thumbsup:

File: server/athenian/api/models/state/README.md (hunk 1 above)
Comment (vmarkovtsev) [reply server/athenian/api/models/state/README.md]: Oops

**Pull request merged**